_posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown

```diff
@@ -85,6 +85,37 @@ using `optional: true`:
 belongs_to :user, optional: true
 ```
 
+- ActiveRecord migrations now need to be tagged with the Rails version they are
+created under. One of the reasons for this is that strings in Rails 4.2 had a
+default size of 4 bytes. In Rails 5.0, their default size is now of 8 bytes.
```

Comment:
@mauro-oto Is it possible to find the PR that introduced this change? That way we could link to it here.

Reply:
@etagwerker I wasn't able to track down the PR that introduced the change, and I checked the Rails source but I couldn't find evidence for it. I found out about this here: https://stackoverflow.com/a/35930912/2754597 which is a post on SO by Rafael Franca. Should I add a link to that or should I omit this explanation?

Reply:
@mauro-oto Please add that link. Thanks!